projects/figure-markets/index.js

```diff
@@ -53,7 +53,8 @@ const getPoolsCollateralValue = async (api) => {
             }
         }
     }))).reduce((acc, cur) => acc += cur || 0, 0)
-    api.add(collateralizedAssets, collateralTotal)
+    // Multiply by 1000 due to decimal conversion in token mapping
+    api.add('FIGR_HELOC', collateralTotal * 1000)
```

Comment:
whats the purpose of this conversion? How many decimals does the token actually have?

Reply:
At the scale of billions of assets the token is only precise to 3 decimal places. This conversion accounts for the pricing precision at Coingecko. @jarrydallison - correct me if I'm wrong.